lib/bap/bap_self.ml

```diff
@@ -108,7 +108,13 @@ module Create() = struct
         Some (Sys.getenv name)
       with Not_found -> None
 
-    let get_param ~(converter:'a converter) ~default ~name =
+    let get_conv : 'a converter -> 'a Arg.converter =
```

Comment:
We have a convention that if a function has a verb in its name, then it has some side effect, e.g., `fetch_data` vs. `data`. So, for functions that just projects a field `x` we use a noun `x`. This is to emphasize a declarative nature of the specification, and that an operation actually doesn't have any const.